File: app/src/main/java/app/islammedia/halaqatalquran/Database/MainDataBase.java
package app.islammedia.halaqatalquran.Database;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import androidx.room.TypeConverters;

@Database(entities = {Student.class , Halaqa.class , HomeWork.class , StudentsAndHomeWorksCrossRef.class}, version = 1 ,exportSchema = false)
@TypeConverters({Converters.class})
public abstract class MainDataBase extends RoomDatabase {
    public abstract MyDAO myDAO();
}

File: app/src/main/java/app/islammedia/halaqatalquran/database/MainDataBase.kt
package app.islammedia.halaqatalquran.database

import androidx.room.Database
import androidx.room.RoomDatabase
import androidx.room.TypeConverters
import app.islammedia.halaqatalquran.database.entities.*

@Database(entities = [Student::class, Halaqa::class, HomeWork::class, StudentsAndHomeWorksCrossRef::class, HalaqatAndHomeWorksCrossRef::class, HalaqatAndStudentsCrossRef::class], version = 2, exportSchema = false)
@TypeConverters(Converters::class)
abstract class MainDataBase : RoomDatabase() {
    abstract fun myDAO(): MyDAO
}